android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -60,8 +63,31 @@ public void requestPermissions(boolean background) {
     }
 
     @ReactMethod
-    public void startTracking() {
-        Radar.startTracking();
+    public void startTracking(ReadableMap optionsMap) {
+        RadarTrackingOptions.Builder options = new RadarTrackingOptions.Builder();
```

Comment:
Looks good. May want to consider moving this logic to `RNRadarUtils`, same pattern as setting places provider.